ansible_collections/arista/cvp/molecule/cv_facts_v3/test_cv_facts_v3.yml

```diff
@@ -0,0 +1,257 @@
+          - CV_FACTS_V3_RESULT.data.cvp_devices[0].deviceStatus == "Registered" # we get this key only with verbose long
+
+    - name: Collect images facts from {{inventory_hostname}}
+      arista.cvp.cv_facts_v3:
+        facts:
+          - images  # collects all images facts
+      register: CV_FACTS_V3_RESULT
+
+    - name: Check cv_facts_v3 result
+      assert:
+        that:
+          - CV_FACTS_V3_RESULT.changed == false
+          - CV_FACTS_V3_RESULT.data.cvp_configlets == []
+          - CV_FACTS_V3_RESULT.data.cvp_containers == []
+          - CV_FACTS_V3_RESULT.data.cvp_devices == []
+          - CV_FACTS_V3_RESULT.data.cvp_tasks == []
+          - CV_FACTS_V3_RESULT.data.cvp_images != []
+
+    - name: Collect images facts from {{inventory_hostname}} filtered by name
+      arista.cvp.cv_facts_v3:
+        facts:
+          - images
+        regexp_filter: "{{IMAGE_NAME}}" # collects specific image facts
+      register: CV_FACTS_V3_RESULT
+
+    - name: Check cv_facts_v3 result
+      assert:
+        that:
+          - CV_FACTS_V3_RESULT.changed == false
+          - CV_FACTS_V3_RESULT.data.cvp_configlets == []
+          - CV_FACTS_V3_RESULT.data.cvp_containers == []
+          - CV_FACTS_V3_RESULT.data.cvp_devices == []
+          - CV_FACTS_V3_RESULT.data.cvp_tasks == []
+          - CV_FACTS_V3_RESULT.data.cvp_images['EOS-4.25.4M.swi'] is defined
+
+    - name: Collect task facts from {{inventory_hostname}}
+      arista.cvp.cv_facts_v3:
+        facts:
+          - tasks
+      register: CV_FACTS_V3_RESULT
+
+    - name: Check cv_facts_v3 result
+      assert:
+        that:
+          - CV_FACTS_V3_RESULT.changed == false
+          - CV_FACTS_V3_RESULT.data.cvp_configlets == []
+          - CV_FACTS_V3_RESULT.data.cvp_containers == []
+          - CV_FACTS_V3_RESULT.data.cvp_devices == []
+          - CV_FACTS_V3_RESULT.data.cvp_images == []
+          - CV_FACTS_V3_RESULT.data.cvp_tasks != []
+          - CV_FACTS_V3_RESULT.data.cvp_tasks.total_tasks is defined
+
+    - name: Collect task facts from {{inventory_hostname}}
+      arista.cvp.cv_facts_v3:
+        facts:
+          - tasks
+        regexp_filter: "{{TASK_STATUS}}" # filtered by status
+        verbose: long
+      register: CV_FACTS_V3_RESULT
+
+    - name: Check cv_facts_v3 result
+      assert:
+        that:
+          - CV_FACTS_V3_RESULT.changed == false
+          - CV_FACTS_V3_RESULT.data.cvp_configlets == []
+          - CV_FACTS_V3_RESULT.data.cvp_containers == []
+          - CV_FACTS_V3_RESULT.data.cvp_devices == []
+          - CV_FACTS_V3_RESULT.data.cvp_images == []
+          - CV_FACTS_V3_RESULT.data.cvp_tasks != []
+          - CV_FACTS_V3_RESULT.data.cvp_tasks.total_tasks is defined
+
+    - name: Collect task facts from {{inventory_hostname}}
+      arista.cvp.cv_facts_v3:
+        facts:
+          - tasks
+        regexp_filter: "{{MOVE_DEVICE_DATA_EXECUTE.devices_moved.taskIds[0]}}" # filtered by task id
```

Comment:
looks like `MOVE_DEVICE_DATA_EXECUTE` doesn't exist and should be `MOVE_DEVICE_RESULT` instead

Reply:
Yeah its by mistake, as the PR #632 got merged now, I will test this PR again and update it